Cubical/HITs/Hopf.agda

```diff
@@ -1,33 +1,387 @@
-{-# OPTIONS --cubical --safe #-}
+{-# OPTIONS --cubical #-}
```

Comment:
Now that all holes are filled, maybe add `--safe` back?

Reply:
Whoops, thanks for pointing that out !